content/en/docs/reference/access-authn-authz/webhook.md

```diff
@@ -164,13 +164,50 @@ Access to non-resource paths are sent as:
 }
 ```
 
+{{< feature-state feature_gate_name="AuthorizeWithSelectors" >}}
+
+With the `AuthorizeWithSelectors` feature enabled, field and label selectors in the request
+are passed to the authorization webhook. The webhook can make authorization decisions
+informed by the scoped field and label selectors, if it wishes.
```

Comment:
IMO this should more strongly encourage webhook authors to read the specifc SAR docs, i.e. CVE-2022-2880.

Reply:
done